src/resources/views/filters/select2_ajax.blade.php

```diff
@@ -140,7 +140,7 @@ class="dropdown {{ Request::get($filter->name)?'active':'' }}">
 			$("li[filter-name={{ $filter->name }}]").on('filter:clear', function(e) {
 				// console.log('select2 filter cleared');
 				$("li[filter-name={{ $filter->name }}]").removeClass('active');
-				$("li[filter-name={{ $filter->name }}] .select2").select2("val", "");
+                $('#filter_{{ $filter->name }}').select2("val", "");
```

Comment:
Can we change all the quotes in this line to either single or double quotes (but not a random mixture of both)?

Reply:
Haha. No, it's perfect, thank you :-) I wish we had this kind of consistency across all Backpack JS, but hey, we'll get there soon enough :-)